src/app/components/listfolder/listfolder.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, Input } from '@angular/core';
+import { Folder } from '../../interfaces/Folder';
+import { Types } from '../../enums/Types';
+@Component({
+  selector: 'app-listfolder',
+  templateUrl: './listfolder.component.html',
+  styleUrls: ['./listfolder.component.scss'],
+})
+export class ListfolderComponent {
+  @Input() data!: Folder;
+
+  folder: string = Types.folder;
+  file: string = Types.file;
```

Comment:
type of this field is `Types`. also, could you please remove type definitions if they can be computed automatically (`folder = Types.folder;` folder property has `Types` type even if you will not specify it)